src/pages/layout/index.tsx

```diff
@@ -29,6 +48,57 @@ export default function ExplorerLayout({children}: LayoutProps) {
             }}
           >
             <Header />
+            {inDev && (
+              <Collapse in={bannerOpen}>
+                <Alert
+                  sx={{
+                    backgroundColor:
+                      theme.palette.mode === "dark" ? "#1E40AF" : "#1D4ED8",
+                    borderRadius: 0,
+                  }}
+                  icon={
+                    <Stack color="#ffffff">
+                      <link
+                        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined"
+                        rel="stylesheet"
+                      />
+                      <span className="material-symbols-outlined">info</span>
+                    </Stack>
+                  }
+                  action={
+                    <Stack direction="row" spacing={3} color={primary[400]}>
+                      <Link href={""} color={primary[400]}>
+                        <Stack direction="row" paddingTop={1} spacing={1}>
+                          <Typography>LEARN MORE</Typography>
+                          <ArrowForwardIosIcon />
+                        </Stack>
+                      </Link>
+                      <Divider
+                        orientation="vertical"
+                        flexItem
+                        variant="middle"
+                        sx={{backgroundColor: "#3B82F6"}}
+                      />
+
+                      <IconButton
+                        aria-label="close"
+                        color="inherit"
+                        size="medium"
+                        onClick={() => {
+                          setBannerOpen(false);
+                        }}
+                      >
+                        <CloseIcon fontSize="inherit" />
+                      </IconButton>
+                    </Stack>
+                  }
+                >
+                  <Typography color="#ffffff">
+                    Power Aptos network. Stake tokens and earn rewards.
+                  </Typography>
```

Comment:
Is this centered? Feels a bit higher or it's just my eyes.

Reply:
its prob cuz of the icon. once we remove it, its in the center now